syncer/db.go

```diff
@@ -196,57 +248,56 @@ func (conn *Conn) executeSQLWithIgnore(tctx *tcontext.Context, ignoreError func(
 	return ret.(int), nil
 }
 
-func (conn *Conn) executeSQL(tctx *tcontext.Context, queries []string, args ...[]interface{}) (int, error) {
+func (conn *DBConn) executeSQL(tctx *tcontext.Context, queries []string, args ...[]interface{}) (int, error) {
 	return conn.executeSQLWithIgnore(tctx, nil, queries, args...)
 }
 
-func createBaseConn(dbCfg config.DBConfig, timeout string, rawDBCfg *baseconn.RawDBConfig) (*baseconn.BaseConn, error) {
-	dbDSN := fmt.Sprintf("%s:%s@tcp(%s:%d)/?charset=utf8mb4&interpolateParams=true&readTimeout=%s&maxAllowedPacket=%d",
-		dbCfg.User, dbCfg.Password, dbCfg.Host, dbCfg.Port, timeout, *dbCfg.MaxAllowedPacket)
-	baseConn, err := baseconn.NewBaseConn(dbDSN, &retry.FiniteRetryStrategy{}, rawDBCfg)
+func createConn(tctx *tcontext.Context, cfg *config.SubTaskConfig, dbCfg config.DBConfig) (*conn.BaseDB, *DBConn, error) {
```

Comment:
can we use ``createConns` to implements this function?

Reply:
yes, after use `createConns` to implements `createConn`, it looks like [this](https://github.com/pingcap/dm/pull/266/files#diff-18d840e544a27bd6ebe280a590e1dbf9R113), please check it

Reply:
I mean we can keep the function `createConn`, and this function use createConns to implements it.
but your change is ok.